### ISSUE TITLE
Add echo of `Transfer-Encoding` header

### ISSUE THREAD

**Issue by 3nol** [mccutchen/go-httpbin#128]
Independent of the support for chunked responses with [issue#72](https://github.com/mccutchen/go-httpbin/issues/72), it would be nice to have the the `Transfer-Encoding` header being returned when the client sends it.

Unless I am missing something, testing the client for sending this request header (i.e. chunking the request), is therefore currently not possible. This feature exists on the original httpbin.org instance by Kenneth Reitz, as evident by calling
```
curl -X POST ... -d '{}'  -H 'Content-Type: application/json' -H 'Transfer-Encoding: chunked'
```
with [httpbin.org/post](https://httpbin.org/post) and [httpbingo.org/post](https://httpbingo.org/post).

**Comment by 3nol** [comment]
Thanks for the fast response! :+1:

**Comment by mccutchen** [comment]
@3nol fix is released in [v2.9.2](https://github.com/mccutchen/go-httpbin/releases/tag/v2.9.2)!
